templates/top_banner.html

```diff
@@ -0,0 +1,41 @@
+{% load i18n %}
+
+{% with banner=app.very_top_banner %}
+    {% if banner %}
+        <style>
+            #very-top-banner {
+                background-color: {{ banner.background_color }};
+                color: {{ banner.color }};
+                border-color: {{ banner.border_color }};
+            }
+        </style>
+
+        <script>
+        document.addEventListener('DOMContentLoaded', () => {
+            const banner = document.getElementById('very-top-banner');
+
+            if (sessionStorage.getItem('hide-very-top-banner') !== null) {
+                banner.remove()
+                document.body.classList.remove('has-top-banner')
+            }
+            else {
+                document.body.classList.add('has-top-banner')
+
+                document.querySelector('#very-top-banner button').addEventListener('click', e => {
+                    banner.remove()
+                    document.body.classList.remove('has-top-banner')
+
+                    if (e.shiftKey) {
+                        sessionStorage.setItem('hide-very-top-banner', true)
+                    }
+                });
+            }
+        });
+        </script>
+
+        <div id="very-top-banner">
+            <span>{{ banner.message }}</span>
+            <button aria-label="{% trans "Fermer" %}" title="{% trans "Avec majuscule, ferme la bannière pour toutes les pages de cet onglet." %}">×</button>
```

Comment:
```suggestion
            <button aria-label="{% trans "Fermer" %}" title="{% trans "Appuyer sur la touche majuscule en cliquant pour que la fermeture persiste sur cet onglet." %}">×</button>
```

(Me semble plus clair.)